app/code/Magento/ConfigurableProduct/Model/ResourceModel/Product/Type/Configurable/StockStatusInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ConfigurableProduct\Model\ResourceModel\Product\Type\Configurable;
+
+/**
+ * Interface StockStatusInterface
+ * @api
```

Comment:
for patch releases, do not mark newly introduced interfaces as api

Reply:
fixed